### DISCUSSION TITLE
STAR settings for including transposable elements reads in RNA-seq datasets

### DISCUSSION THREAD

**Discussion by milcs40** [alexdobin/STAR#1410] [Q&A] [answered]
Hello,

I'm a seasoned wet lab scientist, that has recently turned to bioinformatics.
I have a long standing interest on the analysis of transposable elements (TEs) in species-specific molecular innovation and, recently, on the impact of TE misregulation, in certain pathological contexts.

For this last point, I've started analysing human RNAseq datasets, trying to quantify the number of reads that are derived from TEs. This brings me to my question: Can you advise on the best parameters to use with STAR to include multi-mapped reads and allow a more quantitative approach to work with TE derived reads in human datasets?

Thank you very much,
Miguel

**Comment by alexdobin** [comment, marked as answer]
Hi Miguel,

for quantifying TEs, I would recommend using tools specifically designed for TEs, e.g.:
https://github.com/mhammell-laboratory/TEtranscripts
This tool uses BAM files as input, which can be generated by STAR. Their recommendation for STAR parameters is to increase the number of allowed alignment loci, with  *--winAnchorMultimapNmax 200 --outFilterMultimapNmax 100*.

Cheers
Alex
